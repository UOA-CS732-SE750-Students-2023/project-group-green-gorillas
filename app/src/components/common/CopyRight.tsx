import React from "react";
import { Link, Typography } from "@mui/material";

export const CopyRight = () => {
  return (
    <Typography
      variant="body2"
      color="text.secondary"
      align="center"
      sx={{ mt: 8, mb: 4 }}
    >
      {"Copyright Â© "}
      <Link color="inherit" href="#">
        Retrospective Monster
      </Link>{" "}
      {new Date().getFullYear()}
      {"."}
    </Typography>
  );
};
