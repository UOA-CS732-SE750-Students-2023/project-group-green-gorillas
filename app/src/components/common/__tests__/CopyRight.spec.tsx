import React from "react";
import { render } from "@testing-library/react";
import { CopyRight } from "../CopyRight";

describe("CopyRight", () => {
  beforeAll(() => {
    Date.prototype.getFullYear = jest.fn().mockReturnValue(2000);
  });

  it("should be render correctly", () => {
    const { container } = render(<CopyRight />);

    expect(container).toMatchSnapshot();
  });
});
