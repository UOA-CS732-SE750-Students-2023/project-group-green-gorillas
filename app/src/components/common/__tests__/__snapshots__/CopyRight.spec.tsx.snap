// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyRight should be render correctly 1`] = `
<div>
  <p
    class="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-ijul54-MuiTypography-root"
  >
    Copyright Â© 
    <a
      class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-duj00d-MuiTypography-root-MuiLink-root"
      href="#"
    >
      Retrospective Monster
    </a>
     
    2000
    .
  </p>
</div>
`;
